Recipes/Recipes/Controllers/MainVC.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  MainViewController.swift
+//  Recipes
+//
+//  Created by Nick Nguyen on 2/10/20.
+//  Copyright © 2020 Lambda Inc. All rights reserved.
+//
+
+import UIKit
+
+class MainVC: UIViewController , UISearchBarDelegate
+{
+    private let networkClient = RecipesNetworkClient()
+    
+    var allRecipes: [Recipe] = [] {
+        didSet {
+            filteredRecipes = allRecipes
+        }
+    }
+    
+    
+    var filteredRecipes = [Recipe]() {
+        didSet {
+            recipesTableViewController?.recipes = filteredRecipes
+        }
+    }
+    
+    var recipesTableViewController : RecipesTableVC? {
+        didSet {
+            recipesTableViewController?.recipes = filteredRecipes
+        }
+    }
+    
+  
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "FirstSegue" {
```

Comment:
I recommend coming up with a better name than "FirstSegue"
When working on bigger projects you will need to have more specific names than this.